examples/contract/simple_auction.js

```diff
@@ -1,3 +1,4 @@
+/*global define def*/
```

Comment:
This file is way stale, untested, and unlikely to work. It should be removed or moved where it won't be confused with working files.